definitions/divorce/json/CaseEvent/CaseEvent-general-referral-nonprod.json

```diff
@@ -23,5 +23,17 @@
     "PostConditionState": "GeneralConsiderationComplete",
     "SecurityClassification": "Public",
     "ShowSummary": "Y"
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "GeneralReferralPayment",
```

Comment:
Very small detail, I think we usually start event ids with a small letter though we've broken that a few times already 😅

Reply:
Yeah i was trying to find a standard for it and it didnt seem we had one. Im happy to change it